webui/react/src/ErrorHandler.ts

```diff
@@ -26,6 +26,7 @@ export enum ErrorLevel {
 export enum ErrorType {
   Server = 'server', // internal apis and server errors.
   Auth = 'auth',
+  Deprecated = 'deprecated',
```

Comment:
**question:** looks like we're not using this yet. Is the plan to use this in the near future or this is just a forgotten loose end during PR dev?

Reply:
hmm I go back and forth on it. some of the errors we run into about experiment config could be due to depracated attributes but we could also be more genera land use the Api type

Reply:
I see, in that case I think the more info the better. Meaning knowing that it's deprecation error, separate from API error is a good thing.